guide/principles/principles.md

```diff
@@ -0,0 +1,169 @@
+
+Security is especially important when onboarding people new to Bitcoin where solutions that might be fine for advanced users might backfire for beginners. New users are likely to start by storing small amounts until they get comfortable with self-custody. A scheme suited for storing large amounts of funds is likely overkill and scary to a beginner, so the concept of progressive security is a good idea. This could let a user upgrade their security and private key management scheme as their savings grow. 
+
+A big part of security is also education and awareness, protecting users from bad actors and potentially their own bad behavior. Bad actors may already know what most users don’t, and be aware of new vulnerabilities. It is unrealistic to expect beginners to take in all the knowledge acquired by advanced users of several years in one go, for example while onboarding to a product. We should therefore consider how to continuously educate and level up user awareness of best practices and risks.
+
+Security can also be a feeling. A product that is polished, good looking, easy to use and transparently communicates how it works can help users feel more secure. Especialy compared to one with the exact same security measures that lack these qualities.
+
+
+**Do**
+- Take safeguarding of users funds seriously
+- Provide suitable private key management schemes for beginners
+- Offer progressive security and upgrade paths
+- Build with bad actors in mind
+- Minimize risk of self-inflicted loss from user negligence
+- Continuously educate users on best practice and risks 
+- Reduce attack surface by minimizing use of external code dependencies
+
+**Don't**
+- Blame the user for losing funds
+- Expect beginners to implement best practice backup strategies
+- Underestimate the added *feeling* of security that can come from well polished products
+
+---
+
+## Privacy
+
+> Minimize collection of personal information, and maximize financial transaction privacy
+
+A common misconception of Bitcoin is that it provides complete anonymity and privacy of transactions. Since the blockchain is an unchangeable ledger of all transactions ever made, it is in fact very hard to not have your complete transaction history visible once even a single one of your addresses is connected to you. If Bitcoin is to become viable for a wider audience and daily use we ought to take privacy seriously. This is certainly not to enable or encourage illicit activity, but to protect individual financial privacy. We would not accept our bank to publish our financial transactions publicly.
```

Comment:
I think an example might help to drive home the point here, "What if every time you spent or received cash, all the transaction details were published to your Twitter or Facebook feed for all your friends to see?" to use the example from [bitcoin.org](https://bitcoin.org/en/bitcoin-core/features/privacy). Or if you used bitcoin to pay for your Netflix subscription would you want them to be able to track past and future transactions?

I think a note on how privacy is a vital component underpinning financial freedom and freedom more generally. There is no freedom absent privacy in my opinion.

Reply:
Tweaked example in 7d0ef6e.